examples/shared_samples/enclosure.rb

```diff
@@ -105,9 +152,11 @@
 utilization = item2.utilization(startDate: t)
 puts utilization
 
-puts "\nGetting the script"
-script = item2.script
-puts script
+if @client.api_version < 600
```

Comment:
i don't this endpoint in API500 as well. Can you please confirm?

Reply:
yea it is not for 500 as well